src/main/kotlin/nl/avisi/structurizr/site/generatr/CreateStructurizrWorkspace.kt

```diff
@@ -1,16 +1,25 @@
 package nl.avisi.structurizr.site.generatr
 
+import com.structurizr.Workspace
+import com.structurizr.documentation.Format
+import com.structurizr.documentation.Section
 import com.structurizr.dsl.StructurizrDslParser
-import com.structurizr.export.plantuml.C4PlantUMLExporter
 import com.structurizr.model.Element
 import java.io.File
 
+fun safeCreateStructurizrWorkspace(workspaceFile: File) =
+    try {
+        createStructurizrWorkspace(workspaceFile)
+    } catch (e: Exception) {
+        e.printStackTrace()
+        createLoadingErrorWorkspace(workspaceFile, e)
```

Comment:
Thinking more on this, I think we can do better now with the switch to web sockets. We could pass any creation error using the opened web socket to the browser and show it there in a kind-of modal on the current page. Doing so would remove the need to create an "error" workspace and also lets the browser stay on the current page until the site got successfully generated again.

Reply:
This has been remove in favor of reporting error back via web sockets